src/widgets/hits/hits.tsx

```diff
@@ -151,7 +151,10 @@ const hits: HitsWidget = function hits(widgetOptions) {
     render(null, containerNode)
   );
 
-  return makeHits({ escapeHTML, transformItems });
+  return {
+    ...makeHits({ escapeHTML, transformItems }),
+    $$params: widgetOptions,
```

Comment:
instead we should forward $$params to the widget itself from the makeWidget call:

```
return makeHits({ escapeHTML, transformItems, $$params: widgetOptions });
```

Reply:
Yes that's also a possibility, I will update the implementation to do this.